calendar.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Community Calendar - Editing"
+author: "Graeme Stewart"
+layout: default
+---
+
+The HSF [Community Calendar](https://calendar.google.com/calendar/embed?src=e4v33e1a1drbncdle1n03ahpcs%40group.calendar.google.com&ctz=Europe/Amsterdam)
+can be edited by people in the [HSF Startup Team](https://groups.google.com/forum/#!forum/hep-sf-startup-team)
```

Comment:
I still don't like the idea that we use one group for 2 different purposes. Why not to create (it is so easy) a group `hep-sf-calendar-editors` where we could add people who may be legitimate editors and don't want to be in the startup team list (despite the notification trick). We can then just add this group to the calendar sharing list (any startup team member can do it) and document the usual `+subscribe` trick to subscribe to this group. Additionally this will give us the ability to contact calendard editors if we need to share some information, something not possible if people disabled email notifications...

Reply:
I can't say I deeply care, but if we do this I think the startup team should be able to also edit (I suppose you were proposing that, right). Then the new group is for extra editors (and should be approved subscriptions I think).